apps/accounts/urls.py

```diff
@@ -4,6 +4,6 @@
 
 urlpatterns = [
 
-    url(r'^user/disable$', views.disable_user, name='disable_user'),
+    url(r'^user/disable/$', views.disable_user, name='disable_user'),
```

Comment:
Can you please brief me with the reason for using `/` before `$`. I might be missing some details here.